config.toml

```diff
@@ -152,6 +152,43 @@ baseName = "humans"
 isPlainText = true
 mediaType = "text/plain"
 
+[related]
```

Comment:
This change slightly alters the default [related configuration](https://gohugo.io/content-management/related/).

To ensure 3 blog posts would appear on each page, `authors` was added to provide a boost for posts that included `tags` that haven't been used often.

Also, setting `includeNewer` to true could result in the list of recommended posts updating as we continue to publish blogs.